src/main/webapp/app/apollon-diagrams/apollon-diagram-tutor.component.html

```diff
@@ -67,17 +79,32 @@ <h2>
         </div>
         <div class="col-4 col-xl-3">
             <div *ngIf="invalidError !== ''" class="alert alert-danger" role="alert">{{invalidError}}</div>
-            <div *ngIf="(!selectedEntities && !selectedRelationships) || (selectedEntities?.length === 0 && selectedRelationships?.length === 0)">Select an element to grade the element.</div>
-            <ng-container *ngIf="assessments">
+            <div
+                *ngIf="(!selectedEntities && !selectedRelationships) || (selectedEntities?.length === 0 && selectedRelationships?.length === 0)">
+                Select an element to grade the element.
+            </div>
+            <ng-container>
                 <div *ngFor="let assessment of assessments" [hidden]="!isSelected(assessment.id, assessment.type)">
                     <div class="card">
-                        <div class="card-header">
-                            <h4 class="card-title" *ngIf="assessmentsNames">Score for <span>{{assessmentsNames[assessment.id]['type']}} </span> <code class="text-primary">{{assessmentsNames[assessment.id]['name']}}</code></h4>
+                        <div class="card-header" [class.conflict]="conflicts && conflicts.get(assessment.id)">
+                            <h4 class="card-title"
+                                *ngIf="assessmentsNames">Score for
+                                <span>{{assessmentsNames[assessment.id]['type']}} </span> <code
+                                    class="text-primary">{{assessmentsNames[assessment.id]['name']}}</code></h4>
                         </div>
                         <div class="card-body">
                             <div class="form-group">
                                 <label>Score </label>
                                 <input type="number" step="0.5" [(ngModel)]="assessment.credits" (ngModelChange)="checkScoreBoundaries()">
+                                <div class="form-text text-muted" *ngIf="conflicts && conflicts.get(assessment.id)">
+                                    <small class="header-text row-container conflicting-scores-label">
+                                        Others assessed similar elements with:
```

Comment:
Can you provide a translation?

Reply:
Currently the assessment page has nearly no translations so i wanted to stay consistent with the rest of the page and just add an english version. I would suggest to put a proper translation on our todo list when we rework the assessment editor

Reply:
ok